packages/extensions/cloud/n8n-extension.manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+	"$schema": "https://raw.githubusercontent.com/n8n-io/n8n/refs/heads/master/packages/%40n8n/extension-sdk/schema.json",
+	"name": "cloud",
+	"publisher": "n8n",
+	"displayName": "Cloud",
```

Comment:
This won't ever be displayed for the cloud module or? Do we need a `displayName`?

Reply:
We might show loaded internal modules at some point, but let's keep for now for consistency.